results/server/tags_validation.py

```diff
@@ -0,0 +1,72 @@
+"""
+Validates tags from yaml using pytest
+"""
+import pytest
+import yaml
+import traceback
+
+with open("server/tags.yml") as f:
+    tags_yaml = yaml.load(f, Loader=yaml.FullLoader)
+
+
+class TestTags:
+    def __init__(self, tags):
+        self.tags_yaml = tags_yaml
+        try:
+            self.validate_tags(self.tags_yaml)
+        except Exception:
+            print("Error in tags.yml")
+            print(traceback.format_exc())
+
+        self.run_tests()
+
+    # @pytest.mark.parametrize("tag", tags.keys())
+    # def test_tag(self, tag):
+    #     self.__validate_tags(tags[tag])
+
+    @staticmethod
+    def validate_tags(tags: dict):
+        for key, value in tags.items():
+            if value["greaterThan"] > value["lessThan"]:
+                raise ValueError("greaterThan value must be less than lessThan value")
+            if value["type"] not in ["float", "int", "bool", "string"]:
+                raise ValueError("type must be float, int, bool or string")
+
+    @staticmethod
+    def validate_tags2(tags):
```

Comment:
Is there a need for two validate_tags functions or is this part of the deprecated files?

Reply:
The latter